google/cloud/forseti/common/gcp_type/ke_cluster.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, project_id, name, description, initial_node_count,
         self.server_config = server_config
         self.resource_full_name = resource_full_name
         self._json = raw_json
+        self.as_dict = raw_dict
```

Comment:
nit: can we call this `self._dict` to be consistent with `self._json`?

Reply:
Originally, I did - but the pylintrc reports that as an error, since the property is used outside of the class.  Since that's in the forseti style checker, I figured it shouldn't begin with `_` since it really is a public member.

I'm not attached to the name though - so if something else is better, I'm happy to adjust.

Reply:
I'll change this to number 2